bot/src/commands/add.js

```diff
@@ -27,17 +28,25 @@ const sendRssItems = async (rssItems, channelId, client) => {
   try {
     for (const item of rssItems) {
       const channel = await client.channels.get(channelId);
-      await channel.send(`${item.title}: <${item.link}>`);
+      const message = await channel.send(`${item.title}: <${item.link}>`);
+      await message.react('↘');
     }
   } catch (error) {
     console.error(error);
   }
 };
 
-const sendPinnedMessage = async (msg, channelId, client) => {
+const unsubscribePinnedMessage = async (url, channelId, client) => {
```

Comment:
This reads a bit confusing to me; it sounds like `unsubscribe` is what the function will do. I'd find a name for this specific kind of pinned message and call the function `postWhatever()`.